beacon_node/store/src/metadata.rs

```diff
@@ -204,6 +205,16 @@ impl StoreItem for BlobInfo {
     }
 }
 
+/// Database parameter relevant to data column custody sync. There is only at most a single
+/// `DataColumnCustodyInfo` stored in the db. This record is added to the db when cgc
+/// count changes and is updated incrementally during data column custody backfill. Once custody backfill
+/// is complete the record is removed from the db.
```

Comment:
Description here is out of date (we no longer delete once custody backfill completes).